met-web/src/services/widgetService/index.tsx

```diff
@@ -73,3 +74,22 @@ export const removeWidget = async (engagement_id: number, widget_id: number): Pr
         return Promise.reject(err);
     }
 };
+
+export const sortWidgets = async (engagement_id: number, sort_index: number): Promise<Widget> => {
+    try {
```

Comment:
A bit confused about what will this call perform with the given params. How will the proper widgets sort_index be updated

Reply:
updated with the widget index